helm/tools/grafana-mcp/templates/remotemcpserver.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: kagent.dev/v1alpha2
+kind: RemoteMCPServer
```

Comment:
I know I could have just used the `Service`, but I prefer this because, since the removal of the `ToolServer`, one has to look at controller logs to see if there are any issues with `Service`/`McpServer`-_based_ tooling.

Reply:
I am cognisant that the original `stdio` implementation may have been used to dog-food the KMCP integration, but hopefully a different solution can be found for that.